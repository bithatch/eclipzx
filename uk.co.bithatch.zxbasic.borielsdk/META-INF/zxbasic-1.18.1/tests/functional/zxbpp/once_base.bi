
#include "once.bi"
#include "once.bi"
