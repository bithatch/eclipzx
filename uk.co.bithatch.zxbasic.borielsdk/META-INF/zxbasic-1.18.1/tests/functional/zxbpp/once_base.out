#line 1 "once_base.bi"

#line 1 "once.bi"


DIM a as Ubyte



DIM b as Ubyte
#line 3 "once_base.bi"

